service/src/main/java/uk/nhs/adaptors/gp2gp/mhs/MhsWebClientFilter.java

```diff
@@ -0,0 +1,31 @@
+package uk.nhs.adaptors.gp2gp.mhs;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.ClientResponse;
+import org.springframework.web.reactive.function.client.ExchangeFilterFunction;
+
+import lombok.extern.slf4j.Slf4j;
+import reactor.core.publisher.Mono;
+
+@Slf4j
+@Component
+public class MhsWebClientFilter {
+    public ExchangeFilterFunction errorHandlingFilter() {
+        return ExchangeFilterFunction.ofResponseProcessor(clientResponse -> {
+            clientResponse.statusCode();
+            if (clientResponse.statusCode().equals(HttpStatus.ACCEPTED)) {
+                LOGGER.info("Mhs Outbound Request successful, status code: {}", clientResponse.statusCode());
```

Comment:
Could potentially reuse the filter for all requests, have a parameter of status code and change the logger messages to: request instead of mhs outbound request and for other class using it. Might be overkill I leave I'll leave it up to you :p